test/app.test.js

```diff
@@ -42,6 +110,19 @@ describe('deploy contract ' + contractName, () => {
     const marketId = 'market.' + contractId;
 
 	beforeAll(async () => {
+    const content = await getContent(metadata);
+    console.log('content: ', content);
+
+    // DOES NOT WORK, ERROR `SyntaxError: Unexpected token a in JSON at position 4`
```

Comment:
I get this same error with an invalid API key. This is probably because you're passing `undefined` as the `token`.

I've opened an issue to get this error message fixed https://github.com/ipfs-shipyard/nft.storage/issues/43 and I've added a couple of Node.js examples that may help. https://github.com/ipfs-shipyard/nft.storage/tree/main/client/examples/node.js

Reply:
@alanshaw Thanks for your help. Yes, I didn't even have dotenv configured!
I've fixed that now and verified that the api key value is there.
But when I read either a non-animated jpg file, or an animated gif file that I've added in that folder, it outputs `newBlob:  Blob {}` when I try to inspect what was read https://github.com/near-apps/nft-launcher/pull/1/files#diff-9e405491419ad2461310036abba702cee4f1cc65e9d70c611c3ac3e28a3508e3R57